tests/common.go

```diff
@@ -1519,6 +1558,12 @@ func ValidateApplicationsPurePxctl(contexts []*scheduler.Context) {
 func ValidateApplicationsPureSDK(contexts []*scheduler.Context) {
 	Step("validate applications", func() {
 		for _, ctx := range contexts {
+			if ctx.App.Key == "wordpress" || ctx.App.Key == "nginx-fa-davol" {
+				ctx.SkipMountOptionValidation = false
+			}
+			if ctx.App.Key == "nginx-fa-davol" {
+				ctx.SkipVolumeValidation = true
```

Comment:
Same here, can we remove this hard-coded check? Ideally all apps should pass validation so if it's failing we should figure out why.

Reply:
Done.